src/components/shared/ContextMenu.tsx

```diff
@@ -4,52 +4,52 @@
 
 import { PureComponent } from 'react';
 import type { ComponentProps } from 'react';
-import ReactDOM from 'react-dom';
 import { ContextMenu as ReactContextMenu } from '@firefox-devtools/react-contextmenu';
 
 import './ContextMenu.css';
 
 type Props = ComponentProps<typeof ReactContextMenu>;
 
 export class ContextMenu extends PureComponent<Props> {
-  _contextMenu: any = null;
-  _takeContextMenuRef = (contextMenu: any) => {
-    this._contextMenu = contextMenu;
+  _contextMenuWrapper: HTMLDivElement | null = null;
+  _takeContextMenuWrapperRef = (wrapper: HTMLDivElement | null) => {
+    this._contextMenuWrapper = wrapper;
   };
 
-  _mouseDownHandler(event: Event): void {
+  _mouseDownHandler = (event: Event): void => {
     // This prevents from stealing the focus from where it was.
     event.preventDefault();
-  }
+  };
 
   override componentDidMount() {
-    if (this._contextMenu) {
-      // The context menu component does not expose a reference to its internal
-      // DOM node so using findDOMNode is currently unavoidable.
-      // eslint-disable-next-line react/no-find-dom-node
-      const contextMenuNode = ReactDOM.findDOMNode(this._contextMenu);
-      if (contextMenuNode) {
-        // There's no need to remove this event listener since the component is
-        // never unmounted. Duplicate event listeners will also be discarded
-        // automatically so we don't need to handle that.
-        contextMenuNode.addEventListener('mousedown', this._mouseDownHandler);
-      }
+    if (this._contextMenuWrapper) {
+      // Add the event listener to the wrapper div that we control
+      // There's no need to remove this event listener since the component is
+      // never unmounted. Duplicate event listeners will also be discarded
+      // automatically so we don't need to handle that.
+      this._contextMenuWrapper.addEventListener(
+        'mousedown',
```

Comment:
Since we now have our own element anyway, couldn't we just use <div onMouseDown={this._onMouseDown}>?

Reply:
Ah, lol, yes exactly :D It makes perfect sense as it makes it a lot cleaner! Changed it.